openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -165,6 +166,14 @@ def _serialize_course_block_structure(request, course_block_structure):
     return course_block_structure_serializer.data, block_keys
 
 
+def create_dummy_request(site, user):
```

Comment:
add doc_string for function & its params

Reply:
Done!